Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:lts-slim
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Bundle app source
+COPY . .
+
+# Install deps
+RUN yarn install --immutable-cache
+RUN npm install --location=global serve # to serve the app
+
+# Run the linter
+RUN yarn eslint --max-warnings=0 .
+
+# Build the app
+RUN DISABLE_ESLINT_PLUGIN=true yarn build # already linted above
+
+# Copy app config to root dir
+COPY config/emay.json build/config.json
```

Comment:
Should `emay` be replaced with something non-personal?

Reply:
Definitely, it's there for testing and will be removed soon. It will be replaced by a k8s ConfigMap in the helm chart that is populated with the correct vars for each env (see: BEE).

Reply:
Gotcha, thanks. FYI that we typically open Terra UI PRs in draft status and keep them that way until testing is done. This helps with the automated PR review triaging (as Terra UI is not owned by a particular team) 😊

Reply:
Done! Will reopen when ready.